### PR TITLE
PROFILES-55

### PR DESCRIPTION
**Pull request by seiferma**
This PR addresses [PROFILES-55](https://jira.palladio-simulator.com/browse/PROFILES-55).

### CODE REVIEW

File: bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/EMFProfilesProjectOperation.java [hunk 1]
```diff
@@ -99,167 +90,140 @@ protected void execute(IProgressMonitor monitor) throws CoreException,
 //        subMonitor.subTask("Opening representation");
 //        openRepresentation();
```
Comment (sdkrach) [inline bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/EMFProfilesProjectOperation.java:91]: you should remove these comments as well, as the openRepresentation() is not there anymore.

File: bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                .orElse(null);
+    }
+
+    public static RepresentationDescription findDescription(Viewpoint viewpoint, String descriptionName) {
+        return viewpoint.getOwnedRepresentations()
+                .stream()
+                .filter(d -> descriptionName.equals(d.getName()))
+                .findAny()
+                .orElse(null);
+    }
```
Comment (sdkrach) [inline bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java:39]: Maybe I missed it, but I do not see any code using these methods.

File: bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+    public static URI getRepresentationsURI(IProject project) {
+        final IResource representationResource = project.findMember("representations." + SiriusUtil.SESSION_RESOURCE_EXTENSION);
+        return Optional.ofNullable(representationResource)
+                .map(IResource::getFullPath)
+                .map(IPath::toString)
+                .map(p -> URI.createPlatformResourceURI(p, true))
+                .orElse(null);
```
Comment (sdkrach) [inline bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java:47]: Seeing orElse(null) always makes me feel a little uneasy. IMHO, if this is a condition which can occur, then why not return the optional. If not, throw an exception.

File: bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java (hunk 2 above)
Comment (seiferma) [reply bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java:39]: Correct, they are no longer used after other cleanup steps.

File: bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java (hunk 3 above)
Comment (seiferma) [reply bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/ViewPointUtil.java]: Me too but leaving the code as unsafe as it was before is no solution. I will throw an appropriate exception at least.

File: bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/EMFProfilesProjectOperation.java (hunk 1 above)
Comment (seiferma) [reply bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/EMFProfilesProjectOperation.java]: Right but it is always a trade-off between introducing an additional dependency and reusing code.

**Pull request merged**